Doc/library/curses.rst

```diff
@@ -1017,6 +1017,18 @@ the following methods and attributes:
    Return the character at the given position in the window. The bottom 8 bits are
    the character proper, and upper bits are the attributes.
 
+   .. note::
+
+      ``inch`` only works for ASCII characters.  Use :meth:`in_wch` instead
+      for unicode support.
+
+.. method:: window.in_wch([x, y])
```

Comment:
```suggestion


.. method:: window.in_wch([x, y])
```

We separate methods by 2 blank lines (just check that my suggestion is correct before committing it).